adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <link href='https://fonts.googleapis.com/css?family=Neuton' rel='stylesheet'>
+</head>
+<body>
+    <header>
+        <div class="title">
+            <!-- <img src = "../adopt-a-dog/images/christopher-harris-57366-unsplash.jpg" alt="brown dog  face in the shadows"  alt = "Christoper harris"class = "header-logo">  -->
+            <img alt="adopt a dog white logo with a white dog drawing" src="adoptadog-logo.svg">
+            <h1>Seattle's premiere dog adoption agency</h1>
+        </div>
+        <div class="checkout">
+                <h2 class="shopping-cart" id="cart"><i class="fas fa-shopping-cart"></i>$0</h2>
+                <button class="checkout-button">Checkout</button>
+                <!-- create a page for check out later -->
+            </div> 
+    </header>
+        <nav class="main-nav">
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="dogs.html">Dogs</a></li>
+                <li> <a href="blog.html">Blog</a></li>
+            </ul>
+        </nav>
+        <article>
+            <h2>Here are all the of our dogs!</h2>
+            <div class="grid-container">
+                <article class="dog-tile">
+                    <img alt="Murphy" src="../adopt-a-dog/images/alex-lacayo-756508-unsplash.jpg" onclick=dogInfo(click.img)>
```

Comment:
You're close. I'm glad you're using the same function for all dogs, but what you're passing in isn't quite right. It's ok to just hard code in the dog information and send it to your functions